### ISSUE TITLE
declare_bad_file_replicas returns `Already declared` in the python client

### ISSUE THREAD

**Issue by haozturk** [dmwm/CMSRucio#629]
**Describe the bug**

When I try to do `declare_bad_file_replicas` in the python client, I get `Already declared` response for some files and Rucio doesn't take any action on such files.

**To Reproduce**
I did 
```
from rucio.client import Client
client = Client()

scope = 'cms'
lfn = '/store/mc/RunIISummer20UL18MiniAODv2/ChargedHiggs_HplusTB_HplusToTB_M-2000_TuneCP5_13TeV_amcatnlo_pythia8/MINIAODSIM/106X_upgrade2018_realistic_v16_L1v1-v2/2530000/BB000872-01E6-9E40-946C-FF514197AE0E.root'
rse = 'T2_US_UCSD'

client.declare_bad_file_replicas([{'scope':scope, 'name':lfn, 'rse':rse}], force=True, reason="XXX")
```
on lxplus where rucio has version 1.31.7 and I got `Already declared` and no action was taken on the file. The file was in `A` status before and after I did it.

@dynamic-entropy suggested me to run it on lxplus9  (rucio 32.3.1) with `force=True` and it worked.

When I tried to declare the following replicate bad, I had no issue: 
```
/store/mc/RunIISummer20UL18MiniAODv2/ChargedHiggs_HplusTB_HplusToTB_M-600_TuneCP5_13TeV_amcatnlo_pythia8/MINIAODSIM/106X_upgrade2018_realistic_v16_L1v1-v2/2520000/5A972BEE-4E04-D34C-B60E-506D30A022AB.root
```
at T2_US_Florida

**Expected behavior**
We should be able declare all `A` files bad with this function.

**Related Issues**

https://its.cern.ch/jira/browse/CMSDM-117

**Comment by dynamic-entropy** [comment]
Hello @haozturk ;  Since this is resolved in 1.31.7; can this be closed? 
Or are you expecting something else?

**Comment by haozturk** [comment]
No, @dynamic-entropy we can close this one
